functests/execassets.py

```diff
@@ -34,6 +34,7 @@ class TestAssetCreate(TestCase):
 
     maxDiff = None
 
+    @classmethod
     def setUp(cls):
         with open(environ["TEST_AUTHTOKEN"]) as fd:
```

Comment:
I thought we were going to rename yhis TEST_AUTHTOKEN_FILENAME?

Reply:
Thought that wasn't in the scope of the PR, will update then

Reply:
Updated everywhere